sktime/covariance/covariance.py

```diff
@@ -448,6 +448,8 @@ def fetch_model(self) -> CovarianceModel:
             mean_0 = self._rc.mean_X()
         if self.compute_ctt or self.compute_c0t:
             mean_t = self._rc.mean_Y()
+        if self.reversible:
+            cov_tt = cov_00
```

Comment:
so this overwrites a calculation that might've been done in line 442 above?

Reply:
Well actually it makes little sense to even compute c_tt if reversible is True, as then c_00 = c_tt anyways. I will automatically set it to False when symmetrized is set to true and issue a warning.

Reply:
Thought about it again, I think raising is better in that case. Otherwise there is weird and potentially unpredictable side effect behavior.

Reply:
makes sense to me!